src/Plugin.php

```diff
@@ -355,8 +355,7 @@ public static function loadTextdomain() {
    * @return string
    */
   public static function paginate_links($link) {
-    $link = preg_replace('@/page/1/?(\?|$)@', '$1', $link);
-    return user_trailingslashit($link);
+    return preg_replace('@page/1/?(\?|$)@', '$1', $link);
```

Comment:
@Mauricio-Urrego, why are u reverting the changes you introduced in https://github.com/netzstrategen/wordpress-shop-standards/pull/139/files already?

Reply:
There is an issue with trailingslashit adding an encoded trailing slash to query parameters whenever pagination is used and a filter is active. They accumulate on every page navigation like '?filter_marken=vitra%2F%2F%2F'. I will find a better solution though. This one does not work on all sites which was the original intention of trailingslashit.